hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/http/TestHttpServer.java

```diff
@@ -448,17 +443,15 @@ public List<String> getGroups(String user) throws IOException {
 
     @Override
     public Set<String> getGroupsSet(String user) throws IOException {
-      Set<String> result = new HashSet();
-      result.addAll(mapping.get(user));
-      return result;
+      return new HashSet<>(mapping.get(user));
     }
   }
 
   /**
    * Verify the access for /logs, /stacks, /conf, and /logLevel
    * servlets, when authentication filters are set, but authorization is not
    * enabled.
-   * @throws Exception 
+   * @throws Exception if there is an error during, an exception will be thrown.
```

Comment:
nit: during what?

Reply:
I added this comment because Yetus prompts us about JavaDoc compilation errors. While I included this description, I should focus more on the surrounding code context. I will improve it.